gunicorn/config.py

```diff
@@ -1309,6 +1309,24 @@ class LogConfig(Setting):
     """
 
 
+class LogConfigDict(Setting):
+    name = "logconfig_dict"
+    section = "Logging"
+    cli = ["--log-config-dict"]
+    validator = validate_dict
+    default = {}
```

Comment:
This might be a little bit misleading since we use `glogging.CONFIG_DEFAULTS` as default. Should we document this or keep it as implementation detail?

Reply:
Maybe we should set the default as `None`? Would that be more "unset" / "undefined" / "default" to people?

Maybe it doesn't matter much either way.